pandas/core/arrays/base.py

```diff
@@ -198,17 +247,20 @@ def take(self, indexer, allow_fill=True, fill_value=None):
 
         Examples
         --------
-        Suppose the extension array somehow backed by a NumPy structured array
-        and that the underlying structured array is stored as ``self.data``.
-        Then ``take`` may be written as
+        Suppose the extension array is backed by a NumPy array stored as
+        ``self.data``. Then ``take`` may be written as
 
         .. code-block:: python
 
            def take(self, indexer, allow_fill=True, fill_value=None):
                mask = indexer == -1
                result = self.data.take(indexer)
-               result[mask] = self._fill_value
```

Comment:
hmm, is there a reason we don't define ``_na_value``? (e.g we *do* on indexes) as a property?

Reply:
Originally there was a `self._fill_value`, but because this was not used internally, we left it out for the moment. The discussion about that is a bit spread, but see eg https://github.com/pandas-dev/pandas/pull/19520#issuecomment-364160988 (4th bullet point is about fill_value, and comment below with Tom's answer) and https://github.com/pandas-dev/pandas/pull/19520#issuecomment-366250746 and the comments below that

Reply:
ok that's fine. I think we *do* need / want a way to set this and a property is good, and ``_na_value`` is used elsewhere (eg. in Index)